codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/RuntimeTypes.kt

```diff
@@ -266,12 +266,29 @@ data class RuntimeType(val name: String?, val dependency: RustDependency?, val n
         fun sdkBody(runtimeConfig: RuntimeConfig): RuntimeType =
             RuntimeType("SdkBody", dependency = CargoDependency.SmithyHttp(runtimeConfig), "aws_smithy_http::body")
 
-        fun parseStrict(runtimeConfig: RuntimeConfig) = RuntimeType(
+        fun parseStrictResponse(runtimeConfig: RuntimeConfig) = RuntimeType(
             "ParseStrictResponse",
             dependency = CargoDependency.SmithyHttp(runtimeConfig),
             namespace = "aws_smithy_http::response"
         )
 
+        fun parseHttpRequest(runtimeConfig: RuntimeConfig) = RuntimeType(
+            "ParseHttpRequest",
+            dependency = CargoDependency.SmithyHttpServer(runtimeConfig),
+            namespace = "aws_smithy_http_server::request"
+        )
+
+        fun serializeHttpResponse(runtimeConfig: RuntimeConfig) = RuntimeType(
+            "SerializeHttpResponse",
+            dependency = CargoDependency.SmithyHttpServer(runtimeConfig),
+            namespace = "aws_smithy_http_server::response"
+        )
+
+        fun serializeHttpError(runtimeConfig: RuntimeConfig) = RuntimeType(
+            "SerializeHttpError",
+            dependency = CargoDependency.SmithyHttpServer(runtimeConfig),
+            namespace = "aws_smithy_http_server::response"
+        )
```

Comment:
if you want, you may want to make a companion object in `ServerCodegen` and define those there

Reply:
Ah, makes sense.. I haven't used a companion object yet, but I have seen them used around the code. I'll follow your advice!